.github/workflows/linux.yml

```diff
@@ -0,0 +1,50 @@
+name: Linux
+
+on:
+  push:
+    branches:
+      - master
+      - 'support/*'
+  pull_request: {}
+
+concurrency:
+  group: linux-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  linux:
+    name: Linux
```

Comment:
I'd use `name: ${{ matrix.distro }}` here to change the job names from "Linux / Linux (debian:11)" to simply "Linux / debian:11". For it to be extra pretty, you could also add display names to the matrix like in https://github.com/Icinga/icingadb/blob/master/.github/workflows/sql.yml.